torch/mps/__init__.py

```diff
@@ -1,5 +1,5 @@
 r"""
-This package enables an interface for accessing MPS backend in python
+This package enables an interface for accessing MPS (Metal Performance Shaders) backend in Python. MPS is an acceleration backend for MacOS that utilizes GPU for computations. 
```

Comment:
Please split the description into several lines and add link to https://developer.apple.com/documentation/metalperformanceshaders

Also, it's not limited to MacOS, i.e. one should be able to use the same on iOS as well.

Reply:
@malfet good point about the operating system. I notice the link you provided has a lit of operating systems. I decided to not list the operating systems, as they could change, but instead refer to metal GPUs; this seems more future proof . Let me know if you would prefer a different approach.